save-backend/src/main/kotlin/com/saveourtool/save/backend/service/TestExecutionService.kt

```diff
@@ -212,10 +213,12 @@ class TestExecutionService(private val testExecutionRepository: TestExecutionRep
                     it.expected = testExecDto.expected
                     it.unexpected = testExecDto.unexpected
 
-                    counters.unmatchedChecks += testExecDto.unmatched ?: 0L
-                    counters.matchedChecks += testExecDto.matched ?: 0L
-                    counters.expectedChecks += testExecDto.expected ?: 0L
-                    counters.unexpectedChecks += testExecDto.unexpected ?: 0L
+                    with(counters) {
+                        unmatchedChecks += unmatchedChecks.increaseIfApplicable(testExecDto.unmatched)
```

Comment:
maybe `=` instead of `+=`? also `testExecDto.unmatched` can be not applicable. need to check it instead of `unmatched`

Reply:
Yeah, sure, thanks